keeper/src/crank.rs

```diff
@@ -73,12 +74,37 @@ pub async fn runner(
         futures::future::join_all(handles1),
         futures::future::join_all(handles2),
         futures::future::join_all(handles3),
+        loop_check_new_listings_and_abort(
+            mango_client.clone(),
+            interval_check_new_listings_and_abort
+        ),
         debugging_handle
     );
 
     Ok(())
 }
 
+pub async fn loop_check_new_listings_and_abort(mango_client: Arc<MangoClient>, interval: u64) {
```

Comment:
general thought, not for this PR: we should use rust's Duration type for all of these